### ISSUE TITLE
Couldn't cipher the signature. Maybe YouTube has changed the cipher algorithm. Notify this issue on GitHub: need more than 1 value to unpack

### ISSUE THREAD

**Issue by yeeking** [pytube/pytube#50]
Hello!

The following code generates the error in the subject:

from pytube import YouTube 
yt = YouTube()
yt.url = "http://www.youtube.com/watch?v=J9OLotPnCVY"

---

Traceback (most recent call last):
  File "test.py", line 3, in <module>
    yt.url = "http://www.youtube.com/watch?v=J9OLotPnCVY"
  File "/Users/matthew/src/pyenv/lib/python2.7/site-packages/pytube/api.py", line 95, in url
    self._get_video_info()
  File "/Users/matthew/src/pyenv/lib/python2.7/site-packages/pytube/api.py", line 291, in _get_video_info
    signature = self._cipher(stream_map["s"][i], js_url)
  File "/Users/matthew/src/pyenv/lib/python2.7/site-packages/pytube/api.py", line 356, in _cipher
    "this issue on GitHub: %s" % e)
pytube.exceptions.CipherError: Couldn't cipher the signature. Maybe YouTube has changed the cipher algorithm. Notify this issue on GitHub: need more than 1 value to unpack

**Comment by mtahirtariq** [comment]
Having same issue!!!
To be more specific exception is raised from tinyjs.py's JSVM.compile() method. 

``` python
    regex = re.compile(r"(\w+\.)?(\w+)\(([^)]*)\)")
    code = code.replace("return ", "return=")
    for instruction in code.split(";"):
        #print instruction
        var, method = instruction.split("=")
```

`code` is equal to `u'hG.UN(a,20);hG.tp(a,3);hG.UN(a,37);hG.mf(a,46);return=a.join("")'` and `code.split(";")` generates `[u'hG.UN(a,20)', u'hG.tp(a,3)', u'hG.UN(a,37)', u'hG.mf(a,46)', u'return=a.join("")']` which fails `var, method = instruction.split("=")` because of no `=` in the instructions and outcome is the exception.

**Comment by akatasonov** [comment]
I confirm

Traceback (most recent call last):
  File "download_videos.py", line 10, in <module>
    yt.url = "http://www.youtube.com/watch?v=UPFQ8EJ_DHk"
  File "C:\Python27\lib\site-packages\pytube\api.py", line 95, in url
    self._get_video_info()
  File "C:\Python27\lib\site-packages\pytube\api.py", line 291, in _get_video_info
    signature = self._cipher(stream_map["s"][i], js_url)
  File "C:\Python27\lib\site-packages\pytube\api.py", line 356, in _cipher
    "this issue on GitHub: %s" % e)
pytube.exceptions.CipherError: Couldn't cipher the signature. Maybe YouTube has changed the cipher algorithm. Notify this issue on GitHub: need more than 1 value to unpack

**Comment by nficano** [comment]
Seems to be fixed in 4.0.0.
